src/khoj/routers/api_chat.py

```diff
@@ -564,9 +564,9 @@ async def chat(
     country = body.country or get_country_name_from_timezone(body.timezone)
     country_code = body.country_code or get_country_code_from_timezone(body.timezone)
     timezone = body.timezone
-    image = body.image
+    raw_images = body.images
```

Comment:
Should we / can we enforce any kind of size constraint on this parameter? Doesn't need to be super aggressive, but probably makes sense to bake in some rate limiting.

Reply:
Added API rate limiter to constrain number of images and size of images that the API accepts from clients.